js/src/components/modals/KeyphrasesTable.js

```diff
@@ -0,0 +1,44 @@
+/* External dependencies */
+import React from "react";
+import PropTypes from "prop-types";
+import { Fragment } from "@wordpress/element";
+
+/**
+ * The Related Keyphrases table component.
+ */
+class KeyphrasesTable extends React.Component {
```

Comment:
Can't this component be a stateless component or will it require a state later on?

Reply:
This will be done in a separate issue on Jira

Reply:
For the architects, do we want to make this a stateless component? This has been merged already so we have a basic modal to work with but it might be worthwhile to consider this making this stateless.

Reply:
Yep, makes much more sense as a stateless component. This doesn't seem like a table that is going to need any sorting capabilities or anything that are internal to the component itself. So this is simply a pure component that just needs to render based on its props.